src/taskProvider.ts

```diff
@@ -237,7 +237,7 @@ export class TaskProvider implements vscode.TaskProvider {
     // Runs the app on the speculos emulator for the selected device model, in the docker container.
     const exec = `docker exec -it  ${
       this.containerName
-    } bash -c 'speculos --model ${getSelectedSpeculosModel()} build/${getSelectedSpeculosModel()}/bin/app.elf'`;
+    } bash -c 'speculos --model ${getSelectedSpeculosModel()} bin/app.elf'`;
```

Comment:
I would prefer fixing the directory name for nano s plus instead of just loading from bin directory... otherwise it could lead to some unexpected behavior for the user if he selects a target but doesn't rebuild before launching speculos, the task will try to use `--model <selected model>` with a mismatched binary in `bin`